keylime-agent/src/main.rs

```diff
@@ -149,16 +176,27 @@ async fn main() -> Result<()> {
         None
     };
 
-    let mut measuredboot_ml_path = Path::new(MEASUREDBOOT_ML);
-
-    // Allow setting the binary bios measurements log path when testing
+    // load path for MBA logfile
+    let mut measuredboot_ml_path =
+        Path::new(&config.agent.measuredboot_ml_path);
     let env_mb_path: String;
     #[cfg(feature = "testing")]
     if let Ok(v) = std::env::var("TPM_BINARY_MEASUREMENTS") {
         env_mb_path = v;
         measuredboot_ml_path = Path::new(&env_mb_path);
     }
```

Comment:
This should be dropped, and the tests modified to use the new added way to set the measurements logs.

Note that the tests only passed because this is still in place, as the CI is setting this env var.  Otherwise it would had failed as the resulting path after `config_get_file_path()` would not be the expected one.

Reply:
I'd be very happy to drop this, but not while the CI is using it -- correct?
Unless you want to hold this PR until CI is fixed, we need to leave it in place.

Reply:
This can be fixed in a separate PR.